### ISSUE TITLE
[Bug Report]

### ISSUE THREAD

**Issue by maffe03** [SapuSeven/BetterUntis#138]
<details>
<summary>Logs</summary>

```
Request ID: 14
Error code: -8504
Error message: bad credentials
```
</details>

**Additional information**

- Android version: _9_
- BetterUntis version: _3.4.1 (25)_
- Installation source: _com.google.android.packageinstaller_

**Comment by SapuSeven** [comment]
This has probably been resolved in version 3.4.3. Please update the app and test it again.

**Comment by SapuSeven** [comment]
Possibly related to #112 

Have you tried to update your profile?
